hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/dml/TestDataSkippingQuery.scala

```diff
@@ -113,4 +113,82 @@ class TestDataSkippingQuery extends HoodieSparkSqlTestBase {
       )
     }
   }
+
+  test("bucket index query") {
+    // table bucket prop can not read in query sql now, so need set these conf
```

Comment:
table bucket prop can not be read in the query sql now, so need to set these configs